views.py

```diff
@@ -6,37 +6,40 @@
 from django.core.urlresolvers import reverse
 from django.http import HttpResponseRedirect
 from django.views.generic import TemplateView
-from djangular.views.mixins import allow_remote_invocation, JSONResponseMixin
-from corehq.apps.prelogin.forms import ContactDimagiForm
 from corehq.apps.style.decorators import use_bootstrap3
 
+MAIN_FORM = 'main'
+SUPPLY_FORM = 'supply'
+
+HUBSPOT_PORTAL_IDS = {
+    MAIN_FORM: '503070',
```

Comment:
no, because this repo is not included as part of the main hq and not intended for reuse. It would be private if we could, but these keys are public anyway, so not a problem if they're here